web/channels/user_socket.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Habits.UserSocket do
+  use Phoenix.Socket
+
+  ## Channels
+  channel "notifications", Habits.NotificationChannel
```

Comment:
While it doesn't quite matter since we only let one user register, this should perhaps be tied to the user—e.g. `"notifications:user_id"`—and include some kind of authentication, perhaps with the user's token.